doc-content/shared-kie-docs/src/main/asciidoc/Workbench/RemoteAPI/security-management-rest-api-users-ref.adoc

```diff
@@ -0,0 +1,255 @@
+[id='security-management-rest-api-users-ref_{context}']
+= Users
+
+The Security Management REST API supports the following endpoints for managing users in {CENTRAL}. The Security Management REST API base URL is `\http://SERVER:PORT/{URL_COMPONENT_CENTRAL}/rest/`. All requests require HTTP Basic authentication or token-based authentication for the `admin` user role.
+
+[GET] /users::
++
+--
+Returns all users in {CENTRAL}.
+
+.Example server response (JSON)
+[source,json]
+----
+[
+    "newUser",
+    "user1",
+    "user2",
+]
+----
+--
+
+[GET] /users/{userName}/groups::
++
+--
+Returns all groups assigned to a specified user.
+
+.Request parameters
+[cols="15%,45%,15%,25%", frame="all", options="header"]
+|===
+|Name
+|Description
+|Type
+|Requirement
+
+|`userName`
+|Name of the user for whom you are retrieving assigned groups
+|String
+|Required
+|===
+
+.Example server response (JSON)
+[source,json]
+----
+[{
+		"group1"
+	},
+	{
+		"group2"
+	}
+]
+----
+--
+
+[GET] /users/{userName}/roles::
++
+--
+Returns all roles assigned to a specified user.
+
+.Request parameters
+[cols="15%,45%,15%,25%", frame="all", options="header"]
+|===
+|Name
+|Description
+|Type
+|Requirement
+
+|`userName`
+|Name of the user for whom you are retrieving assigned roles
+|String
+|Required
+|===
+
+.Example server response (JSON)
+[source,json]
+----
+[
+  {
+    "name": "process-admin"
+  },
+  {
+    "name": "manager"
+  },
+  {
+    "name": "admin"
+  }
+]
+----
+--
+
+[POST] /users::
++
+--
+Creates a specified user with specified roles and groups.
+
+.Example request body (JSON)
+[source,json]
+----
+{
+  "name": "newUser",
+  "roles": [
+    "roles.."
+  ],
+  "groups": [
+    "groups.."
+  ]
```

Comment:
Here the `groups...` means that a list of groups should be specified? Maybe we should come up with a list as with other examples.

Reply:
+1, @hmanwani-rh, For roles we can add ["admin", "developer"] and for  groups we can add ["group1", "group2"], since we are showing these in their GET counterparts.

Reply:
Done